convert2rhel/logger.py

```diff
@@ -183,7 +182,7 @@ def format(self, record):
             new_fmt = fmt_orig if self.color_disabled else bcolors.OKGREEN + fmt_orig + bcolors.ENDC
             self._fmt = new_fmt
             self.datefmt = "%m/%d/%Y %H:%M:%S"
-        elif record.levelno in [logging.INFO, LogLevelFile.level]:
+        elif record.levelno in [logging.INFO]:
```

Comment:
Note, after removing this, `self._fmt` and `self.datefmt` will be set by the `else:` clause (the same as `logging.DEBUG`)